infra/feast-operator/internal/controller/services/client.go

```diff
@@ -86,3 +99,35 @@ func (feast *FeastServices) initCaConfigMap() *corev1.ConfigMap {
 	cm.SetGroupVersionKind(corev1.SchemeGroupVersion.WithKind("ConfigMap"))
 	return cm
 }
+
+// GetCustomCertificatesBundle retrieves the custom CA bundle ConfigMap if it exists
+func (feast *FeastServices) GetCustomCertificatesBundle() CustomCertificatesBundle {
```

Comment:
does it make more sense to put this with the rest of the TLS methods instead of client.go?